### PR TITLE
mark subMode as reserved

### PR DESCRIPTION
**Pull request by derhuerst**
As discussed in #4, the way to get a release out soon is to mark `subMode` as reserved for now and specify possible values in the future.

### CODE REVIEW

File: spec/readme.md [hunk 1]
```diff
@@ -221,9 +225,18 @@ The `departureDelay` and `arrivalDelay` fields should only contain a value if re
...
 ## modes
 
-Work in progess, see [the tracking issue for adding new `mode`s](https://github.com/public-transport/friendly-public-transport-format/issues/4).
+As discussed in [#4](https://github.com/public-transport/friendly-public-transport-format/issues/4), we decided to have two fields `mode` and `subMode`.
 
-- `walking`
-- `train` – high-speed, regional, commuter/urban & underground trains
+The following list shows all possible values for a `mode` property. For consumers to be able to use `mode` meaningfully, we will keep this list very short.
+
+- `train`
```
Comment (derhuerst) [inline spec/readme.md]: I would like to have helpful descriptions for these, just like the two I deleted ("urban & long-distance water transport"). @juliuste @kiliankoe please make proposals.

**Comment by derhuerst** [comment]
@juliuste @kiliankoe bump. (:

**Comment by derhuerst** [comment]
@juliuste @kiliankoe bump 2

**Comment by kiliankoe** [comment]
Sorry for not weighing in on this earlier. I'd like to keep my opinion out of this, as you guys seem to have a much better understanding of the intricacies of transport modes and what the different terms imply.

**Comment by derhuerst** [comment]
@kiliankoe Okay. Didn't mean to push you off by sounding too "involved" with public transport. Feel welcome to participate whenever you feel like a point is not being addressed!

**Comment by derhuerst** [comment]
@juliuste bump

**Comment by juliuste** [comment]
LGTM 👍

**Pull request merged**